src/main/java/software/sigma/bu003/internship/vedmid_andrii/entity/Part.java

```diff
@@ -0,0 +1,22 @@
+package software.sigma.bu003.internship.vedmid_andrii.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Data
+@Document
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class Part {
+    @Id
+    private String id;
```

Comment:
I would use a compound id of brand + code, because it;s the easiest way to have unique list of parts...